go/libkb/upak_loader.go

```diff
@@ -36,7 +36,7 @@ type UPAKLoader interface {
 	PutUserToCache(ctx context.Context, user *User) error
 	LoadV2WithKID(ctx context.Context, uid keybase1.UID, kid keybase1.KID) (*keybase1.UserPlusKeysV2AllIncarnations, error)
 	CheckDeviceForUIDAndUsername(ctx context.Context, uid keybase1.UID, did keybase1.DeviceID, n NormalizedUsername, suppressNetworkErrors bool) error
-	Batcher(ctx context.Context, getArg func(int) *LoadUserArg, processResult func(int, *keybase1.UserPlusKeysV2AllIncarnations) error, window int) (err error)
+	Batcher(ctx context.Context, getArg func(int) *LoadUserArg, processResult func(int, *keybase1.UserPlusKeysV2AllIncarnations) error, ignoreError func(err error) bool, window int) (err error)
```

Comment:
If you think it's better I can make this take an options struct or split it out into different functions so callers aren't passing in default values.

Reply:
Eh it's probably fine.. 🤷